test/integration/auth/node_test.go

```diff
@@ -224,13 +224,26 @@ func TestNodeAuthorizer(t *testing.T) {
 
 	createNode2MirrorPod := func(client clientset.Interface) func() error {
 		return func() error {
-			_, err := client.CoreV1().Pods("ns").Create(context.TODO(), &corev1.Pod{
+			const nodeName = "node2"
```

Comment:
out of scope for this PR and more of a nit, but ideally the tests here should have used a more unique node name - `node-<some-uuid>`, because nodes are global and `node2` could be already taken.

Reply:
this is an integration test, it has a clean etcd to work with

Reply:
and the node name relationships are important for all the relational data created in the test